pkg/config/manifest.go

```diff
@@ -59,26 +61,13 @@ type ParameterDefinition struct {
 	Name      string `yaml:"name"`
 	Sensitive bool   `yaml:"sensitive"`
 
-	// These could be swapped out with an inline bundle.ParameterDefinition
-	// from cnab-go, e.g. bundle.ParameterDefinition `yaml:",inline"`
-	Description string    `yaml:"description,omitempty"`
-	Destination *Location `yaml:"destination,omitempty"`
+	// These fields represent a subset of bundle.Parameter as defined in deislabs/cnab-go,
+	// minus the 'Description' field (definition.Schema's will be used)
+	Definition  string    `yaml:"definition"`
```

Comment:
I'm not sure that this field is needed? It's not used in our code anywhere (the definition in the bundle.json is set to the parameter name currently).

Was the intent to allow them to override it with this? I thought at one point we were going to set it to "parameter_foo", "output_foo".

Reply:
Good catch; removed!